helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ResourceMonitor.java

```diff
@@ -422,22 +440,42 @@ public long getMissingReplicaPartitionGauge() {
     return _numLessReplicaPartitions.getValue();
   }
 
+  @Deprecated
```

Comment:
Given the assumption that user will always rely on the MBean Server, I think we can directly remove these methods? They are now only for testing.